scripts/zones/Oldton_Movalpolos/mobs/Bugallug.lua

```diff
@@ -14,6 +14,19 @@ entity.onMobEngaged = function(mob)
 end
 
 entity.onMobDeath = function(mob, player, isKiller)
+    if isKiller then
```

Comment:
You don't need to spread the credit out to the killers party, that happens automatically inside this func: https://github.com/LandSandBoat/server/blob/78f8f36e4c8beaef97bba8648dda9e5ba137b89d/src/map/lua/luautils.cpp#L3231